--- bin/gcovr	2018-05-28 17:41:20.327720803 +0200
+++ bin-fixed/gcovr	2018-05-28 18:12:33.887306599 +0200
@@ -456,7 +456,7 @@
 # Process a single gcov datafile
 #
 def process_gcov_data(data_fname, covdata, source_fname, options):
-    INPUT = open(data_fname, "r")
+    INPUT = open(data_fname, "r", encoding='utf-8')
     #
     # Get the filename
     #
@@ -1716,7 +1716,7 @@
         data['ROWS'] = []
         currdir = os.getcwd()
         os.chdir(root_dir)
-        INPUT = open(data['FILENAME'], 'r')
+        INPUT = open(data['FILENAME'], 'r', encoding='utf-8')
         ctr = 1
         for line in INPUT:
             data['ROWS'].append(
@@ -1725,7 +1725,7 @@
             ctr += 1
         INPUT.close()
         os.chdir(currdir)
-        data['ROWS'] = '\n'.join(data['ROWS'])
+        data['ROWS'] = '\n'.join(data['ROWS'], encoding='utf-8')
 
         htmlString = source_page.substitute(**data)
         OUTPUT = open(cdata._sourcefile, 'w')
